features/action_chain.feature

```diff
@@ -58,20 +58,6 @@ Feature: Test action chaining
     And I click on "Confirm"
     Then I should see a "Action has been successfully added to the Action Chain" text
 
-  Scenario: I add a config file deployment to the action chain
```

Comment:
🕶  We need to create a config channel for that, and  assign the files to the config-channel
Then the cleanup. Since the feature is to test action-chain, i was thinking that we could get rid of that.